### PR TITLE
Tune chromatic

### PR DESCRIPTION
**Pull request by andrewscfc**
Following https://github.com/bbc/simorgh/pull/10866, this PR tunes our snapshots a bit more by:
- Allowing extra time for the images to load on Media Asset Pages
- Increasing diff tolerance on live radio pages to accomodate the live pulse

Testing
======
Merge and see if flakes for associated stories stop occuring

Helpful Links
======
[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/pages/MediaAssetPage/index.stories.jsx [hunk 1]
```diff
@@ -24,6 +24,11 @@ export default {
   Component,
   title: 'Pages/Media Asset Page',
   decorators: [story => <WithTimeMachine>{story()}</WithTimeMachine>],
+  parameters: {
+    chromatic: {
+      delay: 15000,
```
Comment (karinathomasbbc) [inline src/app/pages/MediaAssetPage/index.stories.jsx:29]: Looks like this is still flaky, but I believe this is the longest delay you can set...? Should we consider trying to ignore the related content images on the media asset page?

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/pages/MediaAssetPage/index.stories.jsx (hunk 1 above)
Comment (andrewscfc) [reply src/app/pages/MediaAssetPage/index.stories.jsx:29]: We have another ticket to ignore images entirely, I guess we could just ignore images on selected stories couldn't we?

Comment (karinathomasbbc) [reply src/app/pages/MediaAssetPage/index.stories.jsx:29]: Let's leave the image issue for the other ticket - this is already a vast improvement, and if we only have to approve 1 flaky UI test, that's already better 😸

Comment (andrewscfc) [reply src/app/pages/MediaAssetPage/index.stories.jsx:29]: ~I'll drop a note on the ticket to suggest we try only flakey stories initially~ looks like the ticket already covers it

Comment (andrewscfc) [inline src/app/pages/MediaAssetPage/index.stories.jsx:29]: Hmm this doesn't seem to have helped as the flake is showing up on this PR still; should I revert this change?

**Review by amoore108 — APPROVED** [review]

**Comment by amoore108** [comment]
Think this can be closed @andrewscfc ?

**Comment by andrewscfc** [comment]
> Think this can be closed @andrewscfc ?

happy to close 👍
